atomics/T1098.001/T1098.001.yaml

```diff
@@ -0,0 +1,35 @@
+attack_technique: T1098.001
+display_name: Account Manipulation - Additional Cloud Credentials
+atomic_tests:
+- name: Create AWS Access Key and Secret Key
+  auto_generated_guid: 8822c3b0-d9f9-4daf-a043-491160a31122
+  description: |
+    Adversaries create their own new access and secret keys to programatically interact with AWS environment
+  supported_platforms:
+  - macos
+  - linux
+  cloud_provider:
```

Comment:
remove this new field `cloud_provider`, instead add it to `supported_platforms` and update the code base to support new options